pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -30,6 +30,10 @@ const (
 	PrometheusKindKey = "prometheus"
 )
 
+// ScrapeProtocol represents supported protocol for scraping metrics.
```

Comment:
Let's list the enum values explicitly in the docs:
```suggestion
// ScrapeProtocol represents a protocol used by Prometheus for scraping metrics.
// Supported values are:
// * `OpenMetricsText0.0.1`
// * `OpenMetricsText1.0.0`
// * `PrometheusProto`
// * `PrometheusText0.0.4`
````

Reply:
Thanks for your review. 
Done